Resources/MessageQueue/MQCommunication.py

```diff
@@ -0,0 +1,64 @@
+""" General Message Queue Interface to create Consumers and Producers
+"""
+from DIRAC import gLogger, S_OK
+from DIRAC.Resources.MessageQueue.MQProducer import MQProducer
+from DIRAC.Resources.MessageQueue.MQConsumer import MQConsumer
+from DIRAC.Resources.MessageQueue.MQConnectionManager import MQConnectionManager
+from DIRAC.Resources.MessageQueue.Utilities import getMQParamsFromCS
+from DIRAC.Resources.MessageQueue.Utilities import generateDefaultCallback
+
+connectionManager = MQConnectionManager() #To manage the active MQ connections.
+
+def createConsumer( mqURI, callback = generateDefaultCallback() ):
+  """ Function creates MQConsumer. All parameters are taken
+      from the Configuration Service based on the mqURI
+      value.
+  Args:
+    mqURI(str):Pseudo URI identifing MQ service. It has the following format
+              mqConnection::DestinationType::DestinationName
+              e.g. blabla.cern.ch::Queue::MyQueue1
+    callback: callback function that can be used to process the incoming messages
+  Returns:
+    S_OK/S_ERROR: with the consumer object in S_OK.
+  """
+  result = _setupConnection( mqURI = mqURI, mType = "consumer" )
+  if not result['OK']:
+    gLogger.error( 'Failed to createConsumer:', result['Message'] )
+    return result
+  return S_OK( MQConsumer( mqManager = connectionManager, mqURI  = mqURI, consumerId = result['Value'], callback = callback ) )
+
+def createProducer( mqURI ):
+  """ Function creates MQProducer. All parameters are taken
```

Comment:
This specific form of comments was purposely added here: https://github.com/DIRACGrid/DIRAC/pull/2981